src/useCases/setupNewUser.ts

```diff
@@ -23,7 +25,14 @@ export default async (
   userCreateDetails: any
 ): PromiseResult<newUserSetupResult> => {
   const result = await createUser(connection, currentUser, userCreateDetails)
+  const groupsForCurrentUser = await getUserGroups(connection, [currentUser.username, userCreateDetails.username])
 
+  if (isError(groupsForCurrentUser)) {
+    console.error(groupsForCurrentUser)
+    return groupsForCurrentUser
+  }
+
+  const groupsForNewUser = groupsForCurrentUser.filter((group: any) => userCreateDetails[group.name] === "yes")
```

Comment:
There are a few situations where we could return in case of an error. Might be best to only do the groups operation just before sending the email?

Reply:
Can do.

Reply:
Review comment 1